Setup/UpgradeSchema.php

```diff
@@ -155,8 +155,8 @@ public function upgrade(
             if ($setup->getConnection()->isTableExists('quote') == true) {
                 $setup->getConnection()->changeColumn(
                     $setup->getTable('quote'),
-                    'delivery_options',
                     'myparcel_delivery_options',
+                    'myparcel_myparcel_delivery_options',
```

Comment:
Wat/waarom is dit?

Reply:
stom. Had deze aangepast maar push/pull was niet goed gegaan. Done